test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -68,7 +68,7 @@ def test_dlc_major_version_dockerfiles(image):
     root_dir = os.path.join(dlc_dir, framework, job_type, "docker")
 
     # Skip older FW versions that did not use this versioning scheme
-    references = {"tensorflow2": "2.2.0", "tensorflow1": "1.16.0", "mxnet": "1.7.0", "pytorch": "1.5.0"}
+    references = {"tensorflow2": "2.3.0", "tensorflow1": "1.16.0", "mxnet": "1.7.0", "pytorch": "1.5.0"}
```

Comment:
Will this change be reverted when this PR is merge ready?

Reply:
@arjkesh made these changes as an exception for this tests to pass. I would let him comment on this.

Reply:
We will not revert this - this is the solution until we have changed the 2.2 directory structure in a followup.